ml4floods/preprocess/utils.py

```diff
@@ -25,7 +25,9 @@ def get_list_of_window_slices(
 
     accumulated_list_of_windows = []
     for ifilename in file_names:
-
+        if ".tif.aux.xml" in ifilename:
+            continue
+        
```

Comment:
```suggestion

```
Is this needed? I guess we can just glob files that ends in `.tif` and this will not be needed.